pages/decision-reviews/index.md

```diff
@@ -4,14 +4,13 @@ template: detail-page
 title: VA Decision Reviews And Appeals
 heading: VA decision reviews and appeals
 display_title: VA decision reviews and appeals
-decision_reviews_banner: true
```

Comment:
The code supporting this property is being removed in https://github.com/department-of-veterans-affairs/vets-website/pull/11646, so this is really just a cleanup.